keyboards/1upkeyboards/pi60/info.json

```
@@ -66,8 +66,17 @@
             {"flags": 2, "x": 20, "y": 13}
         ]
     },
+    "community_layouts": [
+        "60_ansi",
+        "60_ansi_split_bs_rshift",
+        "60_ansi_tsangan",
```

add compatible '60_hhkb' layout
```suggestion
        "60_ansi_tsangan",
        "60_hhkb",
```

---

Added in 9e6bc4da78.